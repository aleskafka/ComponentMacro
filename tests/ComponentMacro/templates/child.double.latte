
<Header n:component>
	<li n:class="double" n:child="double-child">LI</li>
</Header>
