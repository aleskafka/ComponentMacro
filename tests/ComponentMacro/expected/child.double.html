
<div class="header">
 	<h1>Hello</h1>
 	<ul class="header-navigation">
 		<li>John</li>
 		<li class="double">LI</li>
 	</ul>
</div>
